package.json

```diff
@@ -45,10 +45,12 @@
     "autoprefixer": "10.4.19",
     "clsx": "^2.0.0",
     "dotenv": "^16.4.5",
+    "doubly-linked-list": "^1.0.6",
```

Comment:
nit: Is this used? I think I may have inadvertently added it in an early commit.